src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    private static final FigureSupplier supplier = new FigureSupplier();
+    private static final int FIGURE_COUNTER = 6;
```

Comment:
to be precise - this is not a `counter`, this is amount/number of figures

```suggestion
    private static final int NUMBER_OF_FIGURES= 6;
```